apps/web/src/blocks/Layout/Layout.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Layout isn't really a block, right? it's a component, potentially.

Let's move it a `components` folder

Reply:
@inadeqtfuturs I could make another folder called `apps/src/globals` in which we can keep all global payload configurations. 

Let me know your thoughts? This way the global components can follow the format of the blocks and have their schema files within the folder rather than in the `apps/payload/globals`.